From a17f4a90b1af174b38b839fc1b5bc0e933ad1725 Mon Sep 17 00:00:00 2001
From: Friedemann Kleint <Friedemann.Kleint@qt.io>
Date: Fri, 4 Sep 2020 09:32:01 +0200
Subject: Bump cmake version to 3.16

Streamline the features list handling in the module
CMakeLists to use the list functionality introduced in cmake 3.3.

Task-number: PYSIDE-1339
Task-number: PYSIDE-904
Change-Id: I184034cae408c815bd80a09016a21fde2969449e
Reviewed-by: Christian Tismer <tismer@stackless.com>
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

(limited to 'CMakeLists.txt')

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d8a43604d..63c415302 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
-cmake_minimum_required(VERSION 3.1)
-cmake_policy(VERSION 3.1)
+cmake_minimum_required(VERSION 3.16)
+cmake_policy(VERSION 3.16)
 
 set(CMAKE_BUILD_TYPE Release CACHE STRING "Build Type")
 
-- 
cgit v1.2.3

